components/CheckoutModal.vue

```diff
@@ -218,7 +218,7 @@ export default {
 }
 </script>
 
-<style scoped>
+<style lang="scss" scoped>
```

Comment:
Is it needed to turn on scss for regular css files? I think there is no need

Reply:
My intention here was to set it up so for future contributors they wouldn't "miss" to set the lang to "scss". I can remove it if you wish. Same for CopyText.vue

Reply:
Good intention, but I prefer languages to be explicitly defined (as the Zen of Python says :D)
If you could, please make it the default language.
I think if contributors require sass features they would anyway see that it is not compiling via sass/unsupported syntax and enable it anyway

Reply:
> Thank you very much for your pull request! Please check the review comments. Also while this passes the linting, it doesn't compile yet:
> 
> ```
>  ERROR  in ./layouts/default.vue?vue&type=style&index=0&lang=scss&                                                                                                   friendly-errors 16:34:02
> 
> Module build failed (from ./node_modules/sass-loader/dist/cjs.js):                                                                                                   friendly-errors 16:34:02
> SassError: Can't find stylesheet to import.
>    ╷
> 31 │ @import "./css/_transition";
>    │         ^^^^^^^^^^^^^^^^^^^
>    ╵
>   layouts/default.vue 31:9  root stylesheet
> ```
> 
> This is because css folder contains two .styl files which need to be migrated too. Also when stylus is fully removed you can run `yarn remove stylus stylus-loader`
> 
> Also sorry if the PR is not finished yet, I just try to reply with a timely review (: Tip: you can mark PR as draft or add it to the title if it is not yet finished. Thank you!

Aah thanks, I missed this. I will look into it :D